python/test/unit/language/test_core_amd.py

```diff
@@ -1128,13 +1128,10 @@ def kernel(X, Z, BLOCK_M: tl.constexpr, BLOCK_N: tl.constexpr, AXIS: tl.constexp
                          [(dtype, shape, perm)
                           # TODO: bfloat16
                           for dtype in ['float16', 'float32']
-                             for shape in [(64, 64), (128, 128)]
```

Comment:
oh I didn't realize (64, 64) was already there as well. Maybe the original skip of the float32 (128,128) was good enough other than maybe adding a comment on why that float32 (128,128) doesn't work due to LDS size.